run_dir/static/js/base.js

```diff
@@ -184,61 +180,45 @@ marked.setOptions({
   smartypants: false
 });
 //Fix for the whole text getting inserted as heading ids and messing up the headings
-marked.Renderer.prototype.heading =function(text,level,raw){
-   return"<h"+level+'>'+text+"</h"+level+">\n"
-};
-function make_project_links(s){
+marked.Renderer.prototype.heading = (text, level, raw) => `<h${level}>${text}</h${level}>\n`;
+
+const make_project_links = (s) => {
   // Searches for P[\d+] and replaces with a link to the project page
   s = s.replace(/([\W])(P[\d]{3,5})(?!\w)/g, '$1<a class="text-decoration-none" href="/project/$2">$2</a>');
 
-  // Searches for FlowCell IDs and replaces with a link (Most complicated regex ever)
-  // - $1 = Captures a non-word character (javascript can't do lookbehind)
-  // - $2 = Matches flowcell date - eg 150505
-  // - $3 = Matches optional flowcell chunk - eg. _D00450_0168
-  // - $4 = Matches remaining flowcell ID - eg. _AC6H3RANXX or _AC6H3RANXX-SDVLKCH
-  // - Not capture lookahead to make sure that were not followed by any more word characters
-  // Replaces with link to flowcell ID without internal chunk
-  // Example: 150505_D00450_0168_AC6H3RANXX links to 150505_AC6H3RANXX
```

Comment:
Please replace this comment as it explains the regex.

Reply:
Oops!